tests/unit/test_databricks.py

```diff
@@ -50,7 +50,16 @@
 @pytest.fixture(scope="session")
 def databricks():
 
-    databricks = Databricks(config_key="databricks-qa-elt")
+    databricks = Databricks(
+        credentials={
+            "host": "https://adb-1930462786844525.5.azuredatabricks.net",
+            "token": "dapic368839ef25987e36fb727353912854c",
```

Comment:
Remove the token pls (and also change it since this one's compromised).

Reply:
It has already been removed in the last commit, and the token is already deactivated from Databricks